addons/resource/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Fields in module 'resource'---
+resource     / resource.calendar        / hours_per_day (float)         : NEW 
+resource     / resource.calendar.attendance / day_period (selection)        : NEW required: required, selection_keys: ['afternoon', 'morning'], req_default: function
```

Comment:
I would filled this depending on the `date_to` field: if before 12:00 a.m, `morning`. `afternoon` elsewhere.

Reply:
and how I put 12:00 a.m in postgres?

Reply:
No need for that in the proper field. It's a float field.